packages/config/test/async.test.js

```diff
@@ -490,4 +490,37 @@ describe('async', () => {
 
         return expect(actual).to.eventually.deep.equal(expected);
     });
+
+    it('should return levels set with custom paths', () => {
+        const bemConfig = config([{
+            levels: [
+                { layer: 'common', path: 'node_modules/lib/common.blocks' },
+                { layer: 'common', path: 'common.blocks' },
```

Comment:
How to get the good `path` value for `{ block: 'button', layer: 'common', tech: 'css' }` cell if we have 2 paths for a `common` layer?

Reply:
dunno, let's talk about it

Reply:
Decided to merge it in as is.